### PR TITLE
GroceryList page to create and view and edit (someday) a grocery list

### PR DESCRIPTION
**Pull request by mfernandez-rally**

### CODE REVIEW

File: src/app/components/groceryLists/groceryChecklistComponent.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import PropTypes from 'prop-types';
+
+import './groceryList.scss';
+
+export default class GroceryList extends React.Component {
+  static propTypes = {
+    groceryList: PropTypes.object
+  };
+
+  constructor(props) {
```
Comment (WadePeterson) [inline src/app/components/groceryLists/groceryChecklistComponent.js]: Is this a placeholder for later?

File: src/app/containers/groceryListsContainer.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    accountId,
+    currentUserId,
+    groceryLists,
+    newGroceryListModalIsOpen: state.groceryList.newGroceryListModalIsOpen,
+    scheduledRecipes,
+    viewType: ViewTypes.MONTH
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
```
Comment (WadePeterson) [inline src/app/containers/groceryListsContainer.js]: You could use the simple object notation for this:
```
const mapDispatchToProps = {
  createNewGroceryList: GroceryListUtils.createGroceryList,
  toggleCreateGroceryListModal: GroceryListActions.toggleCreateGroceryListModal
};
```

**Comment by WadePeterson** [comment]
🍌 🥕 🐔  🗒

File: src/app/containers/groceryListsContainer.js (hunk 2 above)
Comment (mfernandez-rally) [inline src/app/containers/groceryListsContainer.js]: Oh neat!

**Pull request merged**